src/test/rpc/Tx_test.cpp

```diff
@@ -688,6 +730,29 @@ class Tx_test : public beast::unit_test::suite
                             result.second.transaction(),
                             txMeta,
                             tx);
+
+                        // Compare result to result from account_tx
+                        auto mentioned = tx->getMentionedAccounts();
+
+                        if (BEAST_EXPECT(mentioned.size()))
+                        {
+                            auto a = *mentioned.begin();
+                            auto const accountTxResult =
+                                grpcAccountTx(id, b, a);
+
+                            BEAST_EXPECT(accountTxResult.first);
+                            if (accountTxResult.first)
```

Comment:
`BEAST_EXPECT` was put in the condition on line 737 but appears separately here. Could it be moved in?